src/L2ERC20.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {ERC20} from "openzeppelin/token/ERC20/ERC20.sol";
+import {ERC20Votes} from "openzeppelin/token/ERC20/extensions/ERC20Votes.sol";
+import {ERC20Permit} from "openzeppelin/token/ERC20/extensions/ERC20Permit.sol";
+import {IWormhole} from "wormhole/interfaces/IWormhole.sol";
+
+contract L2ERC20 is ERC20Votes {
+  /// @notice The core bridge used to verify messages.
+  IWormhole coreBridge;
```

Comment:
Sorry for the new issue here, but on re-review I'm noticing, this could also be immutable too, right? I guess I don't know enough for sure about the Wormhole architecture to say that for sure, but my assumption is this wouldn't change right?

Reply:
Yes, the core bridge is a proxy contract so the address should be static

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/11/commits/c96b88106bc7134b4687dcc3553ac66fd52aa96e